src/components/UserInfo/UserInfo.tsx

```diff
@@ -1,3 +1,18 @@
 import React from 'react';
+import { User } from '../../types/user';
 
-export const UserInfo: React.FC = () => <>Put the user here</>;
+type Props = {
+  user: User | null;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  if (user !== null) {
```

Comment:
you can shorten it to 
```suggestion
  if (user) {
```